src/commands/translation/get.ts

```diff
@@ -97,6 +105,10 @@ export default class TranslationGet extends BaseCommand<typeof TranslationGet> {
     });
 
     this.log("");
-    ux.styledJSON(JSON.parse(translation.content));
+    if (format === "po") {
```

Comment:
Nit, but I would flip this conditional here to be "if json, ux.styledJSON(...)" which is the logic we are trying to express and will still hold if we were to add another format.